weave/scorers/bedrock_guardrails.py

```diff
@@ -60,7 +60,7 @@ def format_content(self, output: str) -> dict[str, Any]:
         return {"source": self.source, "content": [{"text": {"text": output}}]}
 
     @weave.op
-    def score(self, output: str) -> WeaveScorerResult:
+    def score(self, *, output: str, **kwargs: Any) -> WeaveScorerResult:
```

Comment:
Since these args are wildcarded now, does that mean that you will have to call `score(output=some_val)` instead of just being able to call `score(some_val)`? It looks like the tests were adjusted in this way. We may want to give a notice that this is potentially a breaking change.

Reply:
Yep.  These were always supposed to be wildcarded -- the fact that they weren't initially is a bug that we didn't catch because the types weren't good enough

Reply:
That's pretty entertaining. I remember when I tried this one out I made similar changes to op.py and then saw all the type errors across the codebase and assumed something was wrong. Thanks for picking this up, huge QOL improvement.